core/history_api.php

```diff
@@ -157,6 +157,24 @@ function history_get_events_array( $p_bug_id, $p_user_id = null ) {
 }
 
 /**
+ * Counts the number of changes done by the specified user within specified time window.
+ * @param  integer $p_duration_in_seconds The time window in seconds.
+ * @param  [type]  $p_user_id             The user id or null for logged in user.
+ * @return integer The number of changes done by user in the specified time window.
+ */
+function history_count_recent_events( $p_duration_in_seconds, $p_user_id = null ) {
```

Comment:
Since this retrieves user-specific data, maybe rename it *history_count_user_recent_events* ?